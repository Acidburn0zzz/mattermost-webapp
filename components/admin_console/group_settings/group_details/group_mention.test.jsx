// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import GroupMention from 'components/admin_console/group_settings/group_details/group_mention.jsx';

describe('components/admin_console/group_settings/group_details/GroupMention', () => {
    test('should match snapshot', () => {
        const wrapper = shallow(<GroupMention name='Test'/>);
        expect(wrapper).toMatchSnapshot();
    });
});
