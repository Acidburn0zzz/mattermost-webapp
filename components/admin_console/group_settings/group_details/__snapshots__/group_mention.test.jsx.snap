// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/admin_console/group_settings/group_details/GroupMention should match snapshot 1`] = `
<div
  className="group-profile"
>
  <div
    className="group-profile-field"
  >
    <label>
      <FormattedMessage
        defaultMessage="Mention:"
        id="admin.group_settings.group_details.group_profile_and_settings.groupname.label"
        values={Object {}}
      />
    </label>
    <label>
      <FormattedMessage
        defaultMessage="@{groupname}"
        id="admin.group_settings.group_details.group_profile_and_settings.groupname"
        values={
          Object {
            "groupname": "Test",
          }
        }
      />
    </label>
  </div>
</div>
`;
